.github/workflows/publish-helm-chart.yaml

```diff
@@ -16,15 +20,16 @@ jobs:
     runs-on: ubuntu-20.04
     steps:
       - name: Checkout
-        uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579  # v2.4.0
+        uses: actions/checkout@v2
```

Comment:
@danielpacak  using the SHA commit, is not a good practice anymore?

Reply:
I think we can trust official GitHub Actions to some extent. When I mentioned SHA some time back I was more concerned about GitHub Actions which are not actively developed or are maintained by individuals, e.g. rajatjindal/krew-release-bot or dmnemec/copy_file_to_another_repo_action